examples/utils/migration-tools/package.json

```diff
@@ -36,13 +46,19 @@
 		}
 	},
 	"scripts": {
+		"api-extractor:commonjs": "flub generate entrypoints --outDir ./dist",
 		"api-extractor:esnext": "flub generate entrypoints --outDir ./lib",
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
 		"build:esnext": "tsc --project ./tsconfig.json",
 		"check:are-the-types-wrong": "echo skip attw --pack .",
 		"check:biome": "biome check .",
-		"check:exports": "api-extractor run --config ./api-extractor-lint.json",
```

Comment:
Should be able to delete `api-extractor-lint.json`.
And also `api-extractor.json` that was already unused.

Reply:
Removed in latest